### PR TITLE
upgrade decidim version to 0.22

### PR DESCRIPTION
**Pull request by anaghavl**
#### :tophat: What? Why?
Upgrade decidim to 0.22.0

### CODE REVIEW

**Review by edgarlatorre — APPROVED** [review]
Nice!!! It looks good to me. Just remember that in order for the geocoder to work you need to upgrade it to 1.6.1
Also another problem I had when upgrading decidim-hospitalet was that after merge I had to run the migrations manually.

**Comment by anaghavl** [comment]
> Nice!!! It looks good to me. Just remember that in order for the geocoder to work you need to upgrade it to 1.6.1
> Also another problem I had when upgrading decidim-hospitalet was that after merge I had to run the migrations manually.

Thank you @edgarlatorre 😄

File: spec/system/authorizations_spec.rb [hunk 1]
```diff
@@ -74,8 +74,6 @@ def fill_in_authorization_form
 
         within ".authorizations-list" do
           expect(page).to have_content("El padró")
-          expect(page).not_to have_link("El padró")
-          expect(page).to have_content(I18n.localize(authorization.granted_at, format: :long, locale: :ca))
```
Comment (mrcasals) [inline spec/system/authorizations_spec.rb:78]: Same as in the other PR, why is this removed? 😕

Comment (anaghavl) [reply spec/system/authorizations_spec.rb:78]: They were both failing and I did a little digging to see it was removed in this PR in decidim, https://github.com/decidim/decidim/pull/2311/files

Comment (mrcasals) [reply spec/system/authorizations_spec.rb:78]: But that PR is from like 3 years ago, and released under Decidim v0.9, so there must be something else going on... 😕

File: config/secrets.yml [hunk 2]
```diff
@@ -32,16 +32,19 @@ default: &default
       client_id: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_ID"] %>
       client_secret: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_SECRET"] %>
   geocoder:
-    here_app_id: <%= ENV["HERE_APP_ID"] %>
-    here_app_code: <%= ENV["HERE_APP_CODE"] %>
+    here_api_key: <%= ENV["HERE_API_KEY"] %>
```
Comment (mrcasals) [inline config/secrets.yml:35]: Can you check if this key is available in the server? Otherwise we need to add it before merging this PR

**Review by mrcasals — APPROVED** [review]

**Comment by mrcasals** [comment]
Good job on finding the fix for the test, BTW! 👏 👏 👏
